recipes/pocker/recipe.yaml

```diff
@@ -0,0 +1,65 @@
+context:
+  name: pocker
+  version: "1.18.4"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/p/pocker_tui/pocker_tui-${{ version }}.tar.gz
+  sha256: d83b6bf81964ba721871cb9bb796dfa369f30de0d36e5f25ef5c5bfdfb26f8c4
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
+  number: 0
+  python:
+    entry_points:
+      - pocker = application.main:cli
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - poetry-core
+  run:
+    - python >=${{ python_min }}
+    - docker-py >=7.0.0,<8.0
+    - textual >=1.0.0
+    - pyyaml >=6.0.1
+    - pydantic >=2.7.3
+    - click >=8.1.7
+    - colorama >=0.4.6
+    - yaspin >=3.0.2
+    - packaging >=24.1
+    - textual-plotext >=1.0.0
+    - pyte >=0.8.2
+    - pipx
+    - __unix
```

Comment:
Can you add a bit of context on this, for example with a comment here? For example, I took at step back and I found https://github.com/pommee/Pocker/issues/9#issuecomment-2727459391 . I guess that last change is still not sufficient to run on Windows?